xdist/remote.py

```diff
@@ -251,17 +251,12 @@ def remote_initconfig(option_dict, args):
 
 
 if __name__ == "__channelexec__":
+    import py
+
     channel = channel  # noqa
     workerinput, args, option_dict = channel.receive()
-    importpath = os.getcwd()
-    sys.path.insert(0, importpath)  # XXX only for remote situations
-    os.environ["PYTHONPATH"] = (
```

Comment:
only after archeology ^^

Reply:
Fair enough.

The only archaeology I could find is this one: https://github.com/pytest-dev/pytest-xdist/issues/376#issuecomment-451463327

Reply:
https://github.com/pytest-dev/pytest-xdist/commit/d81ffaa980d540b1dc215533178fec164a1df727

based on this not having it breaks remote workers, and tox is completely unusable to feed those as tox never got that part done